File: Vowel Counter/Vowels.R
vowels <- c("a", "e", "i", "o", "u")

s <- "jhfuhiuahiaho"

v_count <- 0

for (i in strsplit(s, "")[[1]]) {
  if (i %in% vowels) {
    v_count <- v_count + 1
  }
  else {
    next(i)
  }
}

cat("Number of vowels: ", v_count)

File: Vowel Counter/Vowels.py
vowels = ['a', 'e', 'i', 'o', 'u']

s <- "jhfuhiuahiaho"

v_count = 0

for element in s:
    if element in vowels:
        v_count += 1
    else:
        next

print('Number of vowels: ' + str(v_count))
